lib/Notifications/Notifier.php

```diff
@@ -72,10 +77,23 @@ public function prepare(INotification $notification, string $languageCode): INot
 					throw new InvalidArgumentException();
 				}
 
+				// Add button for more information about the IP-address
+				if ($this->config->getAppValue('suspicious_login', 'show_more_info_button', '1') === "1") {
+					$action = $notification->createAction();
+					$label = $l->t('More information ↗');
+					$link = 'https://iplookup.flagfox.net/?ip=' . $suspiciousIp;
+					$action->setLabel($label)
+						->setParsedLabel($label)
+						->setLink($link, IAction::TYPE_WEB)
+						->setPrimary(true);
+					$notification->addParsedAction($action);
+					$additionalText = $l->t('You can get more info by pressing the button which will open https://iplookup.flagfox.net and show info about the suspicious IP-address.');
+				}
+
 				$notification->setParsedSubject(
 					$l->t('New login detected')
 				)->setParsedMessage(
-					$l->t('A new login into your account was detected. The IP address %s was classified as suspicious. If this was you, you can ignore this message. Otherwise you should change your password.', $suspiciousIp)
+					$l->t('A new login into your account was detected. The IP address %s was classified as suspicious. If this was you, you can ignore this message. Otherwise you should change your password.' . ' ' . $additionalText, $suspiciousIp)
```

Comment:
```suggestion
					$l->t('A new login into your account was detected. The IP address %s was classified as suspicious. If this was you, you can ignore this message. Otherwise you should change your password.', $suspiciousIp) . ' ' . $additionalText
```
Otherwise translations won't work

Reply:
Thanks for the hint! is fixed now :)